src/main/js/controls/PairedResult/helpers/helpers.js

```diff
@@ -449,7 +450,15 @@ const processRegions = (graphContext, config, canvasSVG, { key }) => {
             config.shownTargets.indexOf(key) > -1
         );
     } else {
-        hideAllRegions(canvasSVG);
+        if (
+            config.shouldHideAllRegion === false &&
+            config.shownTargets.length > 0 &&
+            areRegionSame(canvasSVG)
+        ) {
+            canvasSVG.selectAll(`.${styles.region}`).attr("aria-hidden", false);
```

Comment:
Could we use `checkAllRegions` function here?

Reply:
I don't think because  we are also checking for config.shownTargets.length > 0 but in checkAllRegion we are checking for config.shownTargets.length > 1 so we can't.